cpp/IceGrid/icebox/README.md

```diff
@@ -7,8 +7,8 @@ Follow these steps to build and run the demo:
 1. Build the client and server applications:
 
     ```shell
-    cmake -B build
-    cmake --build build --config Release
+    cmake -B build -S . -G Ninja
```

Comment:
I don't how this works on Windows. As far as I can tell, you're building debug and currently cmake + IceBox + debug is broken. See #595.

Reply:
I missed updating these instructions to build Release.